website/docs/cli.md

```diff
@@ -50,6 +50,16 @@ Please note that some functionality (for example, anchor links) will not work in
 
 :::
 
+:::info Running on GitHub Codespaces
```

Comment:
ok to add this here, but I'd like the block to be quite generic

It does not apply only to codespace, and this option is also available in other frameworks: CRA, Gatsby, NextJS...

See for example Next.js doc: https://nextjs.org/docs/api-reference/cli#development